misc/theme-cicada/pkg-descr

```diff
@@ -1 +1,5 @@
-The grey-orange theme.
+Cicada Theme 
+Orange Grey Dark
+(c) remic-webdesign
```

Comment:
If you assert copyright without a license we cannot merge this, because nobody will be able to use it.